guide/docs/popular-topics/reactions.mdx

```diff
@@ -1,8 +1,303 @@
+        # Since the list did not return empty and is not a allowed emoji, we remove it
+        await event_message.remove_reaction(emoji=payload.emoji, member=payload.member)
+
+    # Similar behavior can be useful if you want to use reactions as buttons. Since you have to un-react and react again to repeat the effect
+    # This can be usefull if you want the functionality of buttons, but want a more compact look.
+    # but its also a lot of extra api calls compared to components
+    if str(payload.emoji) in button_emojis:
+        # In a proper bot you would do more checks, against message_id most likely.
+        # As theese reactions might normaly be supplied by the bot in the first place
+
+        # Or if the member has the right roles in case the reaction has a moderation function for instance
+        # Otherwise the awesome_function() can end up going off at wrong places
+        await event_message.remove_reaction(
+            emoji=payload.emoji, member=payload.member
+        )  # Remove the reaction
+        awesome_function()
+        await event_channel.send("Done!", delete_after=10.0)
+        # Short message to let the user know it went ok. This is not an interaction so a message response is not strictly needed
+
+    # A very simple reaction role system
+    if str(payload.emoji) in reaction_roles.keys() and payload.message_id in reaction_messages:
+        role_to_apply = bot.get_guild(payload.guild_id).get_role(reaction_roles[str(payload.emoji)])
+        if (
+            role_to_apply and not role_to_apply in payload.member.roles
+        ):  # Check if we actually got a role, then check if the member already has it, if not add it
+            await payload.member.add_roles(role_to_apply)
+
+
+@bot.listen()
+async def on_raw_reaction_remove(payload: disnake.RawReactionActionEvent):
+    if payload.user_id == bot.user.id:
+        return
+    if not payload.guild_id:
+        return  # guild_id is None if its a DM
+
+    # Counterpart to the simple reaction role system
+    if (
+        str(payload.emoji) in reaction_roles.keys() and payload.message_id in reaction_messages
+    ):  # Check that the emoji and message is correct
+        role_to_remove = bot.get_guild(payload.guild_id).get_role(
+            reaction_roles[str(payload.emoji)]
+        )
+        if (
+            role_to_apply and role_to_apply in payload.member.roles
+        ):  # Check if we actually got a role, then check if the member actually has it, then remove it
+            await payload.member.remove_roles(role_to_apply)
+```
+
+### Example using message_command
+
+We could go with a `slash_command` here, but since we will be targeting other messages, it adds a complication because if the message is in a different channel from where the command is executed; the retrieved message will be `None`.
+Using a `message_command` instead side-steps this issue, since the targeted message will be present in the interaction object.
```

Comment:
Slashed the slash_command reference :catsip: in 337719e
Kept the part about target messages being present in ApplicationMessageCommand's interaction 

Fetching messages is a topic that new developers struggle with in general. So this might warrant its own topic in the guides.
Might file an issue and add it later

Reply:
Yeah, message cache/access could potentially have its own page/section.
Usually things can even be optimized with `PartialMessageable` + `PartialMessage`, not even requiring a fetch if all you want is remove a reaction or edit a message without caring about the previous contents.